src/main/java/uk/gov/hmcts/dm/controller/StoredDocumentController.java

```diff
@@ -197,17 +198,16 @@ public ResponseEntity<Void> getBinary(@PathVariable UUID documentId, HttpServlet
             if (toggleConfiguration.isChunking()) {
                 response.reset();
             }
-            logger.warn("IOException streaming response", e);
+            if (e instanceof ClientAbortException) {
```

Comment:
Can we just remove the if else condition please? I do not see any value add in this, as in either case we are still logging the same error as warning. Just with a slight different message.

Reply:
if it is not an ClientAbortException we can log the full stack.